.golangci.yml

```
@@ -0,0 +1,12 @@
+version: 2
+linters:
+  disable-all: true
+  enable:
+    - errcheck
+    - govet
+    - ineffassign
+    - staticcheck
+    - unused
+    - unparam
+run:
+  formatters: ["gci"]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)